driver_test.go

```diff
@@ -1263,6 +1264,48 @@ func testArray(t *testing.T, json bool) {
 	})
 }
 
+func TestVector(t *testing.T) {
+	testVector(t, false)
```

Comment:
We are lacking the test for JSON, right?

Reply:
Added tests for JSON to `old_driver_test.go`.

Reply:
I believe that adding it close to the Arrow tests is better for now.